salt/iaticdfdbackend.sls

```diff
@@ -0,0 +1,260 @@
+## Ubuntu 20 LTS only!
+#
+# Hetzner Oddness: After first install, postgres does not create it's own cluster!
+# Manually run: pg_createcluster 12 main
+# Then restart to make sure it is picked up correctly
+#
+
+{% from 'lib.sls' import createuser, apache, uwsgi, removeapache, removeuwsgi %}
+
+include:
+  - core
+  - apache
+  - uwsgi
+  - letsencrypt
+
+##################################################################### Normal User
+
+{% set user = 'iaticdfdbackend' %}
+{{ createuser(user) }}
+
+##################################################################### Logging User with special access
+{% set user_logs = 'iaticdfdbackend_logs' %}
+{{ createuser(user_logs) }}
+
+add_{{ user_logs }}_to_group_adm:
+  group.present:
+    - name: adm
+    - addusers:
+      - {{ user_logs }}
+
+##################################################################### Ubuntu Dependencies
+
+iaticdfdbackend-deps:
+    apache_module.enabled:
+      - name: proxy proxy_uwsgi rewrite
+      - watch_in:
+        - service: apache2
+    pkg.installed:
+      - pkgs:
+        - libapache2-mod-proxy-uwsgi
+        - python3-pip
+        - python3-virtualenv
+        - uwsgi-plugin-python3
+        #- redis-server
```

Comment:
Don't be afraid to delete comments like this - leaving this in might make me think later "wait, is/was Redis required for this app?" (and from  my understanding it's not?)